idaes/power_generation/carbon_capture/mea_solvent_system/unit_models/column.py

```diff
@@ -0,0 +1,1799 @@
+        def E3_eqn(blk, t, x):
+            if x == self.liquid_phase.length_domain.last():
+                return Constraint.Skip
+            else:
+                return 1 - blk.E[t, x] <= 0.0
+
+        # ---------------------------------------------------------------------
+        # Boundary Conditions
+        # Enthalpy balance
+
+        @self.Constraint(self.t,
+                         doc='Boundary condition for vapor temperature @ inlet ')
+        def vap_BC_temperature(blk, t):
+            return blk.vap_in_temperature[t] ==\
+                blk.vapor_phase.properties[t, 0].temperature
+
+        @self.Constraint(self.t,
+                         doc='Boundary condition for liquid temperature @ inlet ')
+        def liq_BC_temperature(blk, t):
+            return blk.liq_in_temperature[t] ==\
+                blk.liquid_phase.properties[t, 1].temperature
+
+        # Mass balance-mole_frac
+        @self.Constraint(self.t, self.liq_comp,
+                         doc='Boundary condition for liquid mole fraction @ inlet ')
+        def liq_BC_mole_frac(blk, t, j):
+            return blk.liq_in_mole_frac[t, j] ==\
+                blk.liquid_phase.properties[t, 1].mole_frac_comp[j]
+
+        @self.Constraint(self.t, self.vap_comp,
+                         doc='Boundary condition for vapor mole fraction @ inlet ')
+        def vap_BC_mole_frac(blk, t, j):
+            return blk.vap_in_mole_frac[t, j] ==\
+                blk.vapor_phase.properties[t, 0].mole_frac_comp[j]
+
+        # Mass balance-flow-mol
+        @self.Constraint(self.t,
+                         doc='Boundary condition for vapor flow @ inlet ')
+        def vap_BC_flow_mol(blk, t):
+            return blk.vap_in_flow[t] ==\
+                blk.vapor_phase.properties[t, 0].flow_mol
+
+        @self.Constraint(self.t,
+                         doc='Boundary condition for liquid flow @ inlet ')
+        def liq_BC_flow_mol(blk, t):
+            return blk.liq_in_flow[t] ==\
+                blk.liquid_phase.properties[t, 1].flow_mol
+
+        # Pressure balance
+        @self.Constraint(self.t,
+                         doc='Boundary condition for  pressure @ inlet ')
+        def vap_BC_pressure(blk, t):
+            return blk.bot_pressure[t] ==\
+                blk.vapor_phase.properties[t, 0].pressure
+
+    #==========================================================================
+    # Model initialization routine
+    def initialize(blk, method='default', *args, **kwargs):
+        """
+        Initialize the mea column.
+
+        Args:
+            method: Method of initialization to use in {**'default'** -
+                bootstrap initilization, **json** - reload a saved model state
+                from a json file
+            *args: Positional args fed to the selected initialization function,
+            **kwargs: keyword arguments fed to selected initialization function}
+        """
+        if method == 'default':
```

Comment:
I have updated this to have separate default and load_json methods. Since you can only load a saved JSON file. I am only supporting the current saved file for each process type.